monsteer/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@ set(MONSTEER_LINK_LIBRARIES PUBLIC Brion Lunchbox zeq)
 
 common_library(Monsteer)
 
+add_subdirectory(brionplugin)
```

Comment:
That's fine as well. It should just be consistent across projects. I find brionplugin slightly to long/redundant.